scripts/setup_dev_ubuntu.sh

```diff
@@ -6,12 +6,17 @@
 set -xeuo pipefail
 REPO_ROOT="$(cd "$(dirname "$0")/.."; pwd)"
 
-EXPECTED_RELEASE="bionic|focal"
-RELEASE=$(lsb_release --codename | awk '{print $2}')
+EXPECTED_RELEASE="20|18|16"
+EXPECTED_RELEASE_NAMES="focal|bionic|xenial"
+RELEASE_MAJOR=$(lsb_release --release --short | cut -d . -f 1)
+RELEASE_NAME=$(lsb_release --codename | awk '{print $2}')
 
 if echo "${RELEASE}" | grep -q -e "${EXPECTED_RELEASE}" > /dev/null;
```

Comment:
This test is wrong now since you removed `RELEASE`. It's a good idea to test the failing case not just the successful one.

I would recommend just using a `case` statement. My use of `grep` was a bad idea I think. Best to get rid of that.

Reply:
yikes!

Maybe I'll look into adding shellcheck in our lint process.